manager/integration/framework/node_exec/node_exec.py

```diff
@@ -0,0 +1,116 @@
+from kubernetes import client
+from kubernetes.stream import stream
+
+import time
+
+DEFAULT_POD_TIMEOUT = 180
+DEFAULT_POD_INTERVAL = 1
+
+class NodeExec:
+
+    _instance = None
+    node_exec_pod = {}
+
+    def __new__(cls, *args, **kwargs):
+        if cls._instance is None:
+            cls._instance = super().__new__(cls)
+        return cls._instance
+
+    def __init__(cls):
+        cls.core_api = client.CoreV1Api()
```

Comment:
Suggest having a super class to have these common constructor for instance or class, so no need to repeat this kind of similar code in each subclass.

Reply:
as per https://github.com/longhorn/longhorn-tests/pull/1125#discussion_r1136475552, these classes don't need to be singleton anymore.